packages/dart/sshnoports/bundles/shell/install.sh

```diff
@@ -114,7 +114,7 @@ usage() {
 setup_authorized_keys() {
   mkdir -p "$user_ssh_dir"
   touch "$user_ssh_dir/authorized_keys"
-  chown $user:$user "$user_ssh_dir/authorized_keys" || chown $user "$user_ssh_dir/authorized_keys"
+  chown $user:$user "$user_ssh_dir/authorized_keys" 2>/dev/null || chown $user "$user_ssh_dir/authorized_keys" 2>/dev/null
```

Comment:
suppress errors to chown in all cases, we try as many combinations as possible, we do our best to guess the group name, but it can be different on different platforms. Setting just the user ownership is good enough in most cases.